modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/GridDiscoveryManager.java

```diff
@@ -715,6 +715,9 @@ else if (customMsg instanceof ChangeGlobalStateFinishMessage) {
                     ctx.cache().onDiscoveryEvent(type, customMsg, node, nextTopVer, ctx.state().clusterState());
                 }
 
+                if (type == EVT_DISCOVERY_CUSTOM_EVT)
```

Comment:
Just include it into the next `if` clause.

Reply:
Fixed